Dockerfile.arm64

```diff
@@ -0,0 +1,26 @@
+#
+# Cilium incremental build. Should be fast given builder-deps is up-to-date!
+#
+# cilium-builder tag is the Git SHA of the compatible build image commit.
+# Keeping the old images available will allow older versions to be built
+# while allowing the new versions to make changes that are not backwards compatible.
+#
+#FROM quay.io/cilium/cilium-envoy-builder:c31482c3e49670980c05cafc914320f7949b266f-arm64 as builder
+FROM quay.io/cilium/cilium-envoy-builder:latest-arm64 as builder
```

Comment:
@TrevorTaoARM Can you create a `Dockerfile.multi-arch` that could accept an ARCH as an argument? Later on we would deprecate `Dockerfile` and replace it with the `Dockerfile.multi-arch`

Reply:
@aanm Yes, I would do it also.